examples/plot_glm_retinal_ganglia.py

```diff
@@ -0,0 +1,289 @@
+# -*- coding: utf-8 -*-
+"""
+=================================================================
+GLM for Spike Trains Prediction in Primate Retinal Ganglion Cells
+=================================================================
+
+* Original tutorial adapted from Johnathan Pillow, Princeton University
+* Dataset provided by E.J. Chichilnisky, Stanford University
+* The dataset is granted by the original authors for educational use only
+* Please contact ``pillow@princeton.edu`` if using beyond its purposes
+
+First of all, we would like to thank Professor Johnathan Pillow
+and Professor E.J. Chichilnisky for sharing their tutorial dataset.
+The original MATLAB and Python tutorial can be found from
+https://github.com/pillowlab/GLMspiketraintutorial.
+
+These data were collected by Valerie Uzzell in the lab of
+E.J. Chichilnisky at the Salk Institute. For full information see
+Uzzell et al. (J Neurophys 04) [1]_, or Pillow et al. (J Neurosci 2005) [2]_.
+
+In this tutorial, we will demonstrate how to fit linear GLM and
+linear-nonlinear GLM (a.k.a Poisson GLM) to predict the spike counts
```

Comment:
just say Poisson GLM, no? Not to confuse people with terminology

```suggestion
Poisson GLM to predict the spike counts
```

also I didn't quite get the point of this comparison? Is it show that one of them performs better?

Reply:
Yes, the original purpose is to show which one performs better by measure R^2 .